express/Question1

```diff
@@ -0,0 +1,20 @@
+var express = require('express')
+//get express module
+var path = require('path')
+//get path module
+var app = express()
+//define express as an app
+app.use(express.static(process.argv[3] || path.join(__dirname, 'public')))
+//use app via express to serve static files ** Could you explain the remaining syntax?
```

Comment:
`process.argv` is an array of arguments passed on the command line. `[3]` is the fourth item in the array. The first two would be `node` and the name of the `.js` file. This statement allows passing a directory to the app from the command line.

The CLI-specified path, if undefined, falls through to a default value. `path.join(__dirname, 'public')` creates an absolute path for the directory called `public` based on the currently executing JavaScript file (`__dirname`).